onboard/cv/config.h.in

```diff
@@ -1,7 +1,9 @@
 #pragma once
-#mesondefine TB_DETECTION
+#mesondefine AR_DETECTION
```

Comment:
**Nit:** group these more nicely, i.e. have `AR_DETECTION` followed by `AR_RECORD` followed by `OBSTACLE_DETECTION` followed by `OBS_RECORD`. Just so it's clear in this file what actually matters, in case we have to mess w these late on.

Reply:
Alrighty